### PR TITLE
Added bottom margin to Resume.tsx

### PR DESCRIPTION
**Pull request by SaranshBangar**
Added "mb-8" to kunalkeshan.dev/components/about/Resume.tsx/(ul tag)

Earlier ->
image

Resolved ->
image

Resolves issue #148

### CODE REVIEW

**Comment by SaranshBangar** [comment]
Resolves #148

File: components/about/Resume.tsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ const Resume: React.FC<ResumePageProps> = (props) => {
         Take a look at my{" "}
         <span className="leading-ti bg-portfolio-main px-1">resume</span>
       </h2>
-      <ul className="mx-auto mt-8 flex w-full flex-col items-center justify-center gap-8">
+      <ul className="mx-auto mt-8 mb-8 flex w-full flex-col items-center justify-center gap-8">
```
Comment (kunalkeshan) [inline components/about/Resume.tsx:75]: Hey!! Thanks for pointing this out, you have a sharp eye 🕵️ .
In most pages I've maintained a pattern of keeping the margin-bottom twice the size of the margin-top.
Can you update this same here from `mb-8` to `mb-16`?

**Review by kunalkeshan — CHANGES_REQUESTED** [review]

File: components/about/Resume.tsx (hunk 1 above)
Comment (SaranshBangar) [reply components/about/Resume.tsx:75]: Sure, give me a few mins and I will update that

Comment (kunalkeshan) [reply components/about/Resume.tsx:75]: Awesome, thanks, buddy.

**Comment by SaranshBangar** [comment]
Updated the bottom margin to mb-16

**Review by kunalkeshan — APPROVED** [review]
LGTM! 🚀

**Pull request merged**